src/pages/openPRs.tsx

```diff
@@ -0,0 +1,42 @@
+import { FunctionComponent, useState, useEffect } from 'react';
+import axios from 'axios';
+import Layout from '../components/Layout';
+import Navbar from '../components/navbar';
+import PullRequest from '../components/pullRequests';
+
+const openPRs: FunctionComponent = () => {
+    axios.defaults.baseURL = 'https://staging-api.realdevsquad.com/';
+
+    const [state, updateState] = useState(null);
+    useEffect(() => {
+        axios.get('/pullrequests/open')
+            .then((response) => {
+                updateState(response.data.pullRequests);
```

Comment:
Since the initial value of the state is `null`, wouldn't it make more sense to call it `setState` instead of `updateState`?

`updateState` would have made sense if we were modifying some existing value.

Let me know what you think about it 😄

Reply:
I think both names are fine(updateState or setState). we can think of it as modifying state value from it's previous value(even it is null initially).